# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/src/mediapipeline/MediaPipeline.h
# Commit: c414d82a5325
# Full Hash: c414d82a532508523d40ce63111cb30b2a1076be
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-02-22 05:26:22
# Regressor Bug: 1521879
# File Overlap Count: 1
# Description:
#   Bug 1521879 - Part 1: IPC-based MediaTransport implementation r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D17273
# ==============================================================================

diff -r 3a182b05e2c1 -r c414d82a5325 media/webrtc/signaling/src/mediapipeline/MediaPipeline.h
--- a/media/webrtc/signaling/src/mediapipeline/MediaPipeline.h	Thu Feb 21 19:01:32 2019 +0200
+++ b/media/webrtc/signaling/src/mediapipeline/MediaPipeline.h	Thu Feb 21 16:42:12 2019 +0000
@@ -197,7 +197,7 @@
   virtual void OnRtpPacketReceived(){};
   void IncrementRtcpPacketsReceived();
 
-  virtual void SendPacket(MediaPacket& packet);
+  virtual void SendPacket(MediaPacket&& packet);
 
   // Process slots on transports
   void RtpStateChange(const std::string& aTransportId, TransportLayer::State);